packages/webpack/src/utils/with-web.ts

```diff
@@ -176,7 +176,7 @@ export function withWeb() {
             loader: require.resolve('sass-loader'),
             options: {
               implementation: require('sass'),
-              sourceMap: options.sourceMap,
+              sourceMap: !!options.sourceMap,
```

Comment:
This solution is already being used below https://github.com/nrwl/nx/blob/c134efe33ffb0dcdf24e49909909b696fb5c1c76/packages/webpack/src/utils/with-web.ts#L424